apps/meteor/client/views/admin/customUserStatus/AddCustomUserStatus.tsx

```diff
@@ -27,11 +33,11 @@ function AddCustomUserStatus({ goToNew, close, onChange, ...props }) {
 			goToNew(result)();
 			onChange();
 		} catch (error) {
-			dispatchToastMessage({ type: 'error', message: error });
+			dispatchToastMessage({ type: 'error', message: String(error) });
```

Comment:
```suggestion
			dispatchToastMessage({ type: 'error', message: error });
```

String(error) would just give '[object] Object]' ([moreinfo](https://tc39.es/ecma262/multipage/abstract-operations.html#sec-ordinarytoprimitive), [moreinfo2](https://stackoverflow.com/questions/2485632/valueof-vs-tostring-in-javascript#:~:text=If%20the%20hint%20is%20String,other%20as%20the%20second%20choice.)). If you've been getting undefined in the toast - [this](https://github.com/RocketChat/Rocket.Chat/pull/24773) pr will fix that.